content/docs/setup/kubernetes/upgrading-istio/index.md

```diff
@@ -255,3 +255,63 @@ spec:
     tls:
       mode: DISABLE
 {{< /text >}}
+
+## Migrating the `RbacConfig` to `ClusterRbacConfig`
+
+If you use [Istio authorization](/docs/concepts/security/#authorization) with custom resource of kind `RbacConfig`,
+you need to replace the custom resource of kind `RbacConfig` with kind `ClusterRbacConfig`.
+The rbac config has an implementation bug that causes it to be namespace scoped in some cases. The cluster rbac config
```

Comment:
Is it worth describing the bug in more detail or at least providing a reference to one?

Reply:
I added a link to the bug.